output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/MainTest.java

```diff
@@ -347,15 +349,20 @@ public void tooManyCorruptedSource_invalidTest() throws Exception {
     int exitCode = SystemLambda.catchSystemExit(() -> Main.main(argv));
 
     assertThat(exitCode).isEqualTo(Main.TOO_MANY_CORRUPTED_SOURCES_EXIT_CODE);
-    assertThat(
-            retrieveReportString(
-                resolve(
-                    NEW_NOTICES_TYPE_FOLDER_NAME,
-                    ACCEPTANCE_TEST_REPORT_FOLDER_NAME,
-                    SOURCES_CORRUPTION_REPORT_JSON)))
-        .isEqualTo(
+    MatcherAssert.assertThat(
+        retrieveReportString(
+          resolve(
+            NEW_NOTICES_TYPE_FOLDER_NAME,
+            ACCEPTANCE_TEST_REPORT_FOLDER_NAME,
+            SOURCES_CORRUPTION_REPORT_JSON)),
+        Matchers.anyOf(
```

Comment:
I'm okay with the code. Did you encountered a use case where the order of `source-id-5` and `source-id-4` is reversed? I tried to run the original test after reversing the order and the test is failing.

Reply:
Oh! I just realized your explanation was in #1086. Sorry about that. I am unable to reproduce the bug. Could you provide before and after screenshots in this PR to document the fix?